reference/src/transforms/window.md

```diff
@@ -0,0 +1,56 @@
+# Window
+
+Applies a pipeline to overlapping segments of rows.
+
+```prql_no_test
+window rows:.. range:.. expanding:false rolling:0 {pipeline}
+```
+
+For each row in result, its input segment is determined either by:
+
+- `rows`, which takes a range of rows relative to current row,
+- `range`, which takes a range of values relative to current row value.
+
+Start of the range is inclusive, end of range is exclusive. Index 0 references
+current row. If a bound is omitted, segment will extend until the end of the table (or group).
+
+For example:
+- `rows:0..3`   means current row plus two following,
+- `rows:-3..0`  means two preceding rows plus current row,
+- `rows:-3..5`  means two preceding rows plus current row plus four following rows,
+- `rows:..1`    means all rows from the start of the table to and including current row,
+- `rows:0..`    means current row and all following rows until the end of the table,
+- `rows:..`     means all rows, which same as not having window at all.
+
+> Note: currently, negative integer literals (`-3`) are not implemented.
+
+<!-- TODO: rows vs range example, with visualization -->
+
+For ease of use, there are two flags that override `rows` or `range`:
+
+- `expanding:true` is an alias for `range:..1`. Sum using this window is also known as "cumulative sum".
```

Comment:
Wouldn't this be (but probably I'm misunderstanding):

```suggestion
- `expanding:true` is an alias for `rows:..0`. Sum using this window is also known as "cumulative sum".
```

?

Reply:
@aljazerzen just wanted to check you saw this. IIUC this only matters when there are duplicate values; otherwise they're the same (or possibly my SQL knowledge is rusty)